MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -3755,24 +3755,18 @@ public Money getPayRoll() {
     }
 
     public Money getPayRoll(boolean noInfantry) {
-        if(!campaignOptions.payForSalaries()) {
+        if (getCampaignOptions().payForSalaries()) {
```

Comment:
This one seems like an "if it ain't broke don't fix it" scenario.

Reply:
See the next one for my reasoning on the change.